### PR TITLE
[4.x] Chronos 2.0

### PR DESCRIPTION
**Pull request by sojeda**
https://github.com/laravel/horizon/issues/823

### CODE REVIEW

**Comment by driesvints** [comment]
I guess it's fine to do this since the tests are passing.

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "ext-json": "*",
         "ext-pcntl": "*",
         "ext-posix": "*",
-        "cakephp/chronos": "^1.0",
+        "cakephp/chronos": "^1.0|^2.0",
```
Comment (driesvints) [inline composer.json:17]: Why did you change this later on? I think the bump to just `^2.0` is perfectly fine.

Comment (sojeda) [reply composer.json:17]: In case there are projects that use version 1

Comment (driesvints) [reply composer.json:17]: Those projects need to define the chronos dependency in their own composer.json is they depend on it directly. We're changing the dependency for Horizon here.

Comment (sojeda) [reply composer.json:17]: Ok, I leave it like it was before

**Pull request merged**